SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -22,10 +23,16 @@ SpatialGDK::ComponentUpdate ToComponentUpdate(FWorkerComponentUpdate* Update)
 
 } // anonymous namespace
 
+void USpatialWorkerConnection::SetEventTracer(SpatialGDK::SpatialEventTracer* InEventTracer)
```

Comment:
This doesn't seem like something that should be dynamic. Why not do this when we add in the connection?

Reply:
When do we "add in the connection"?

Reply:
This is only ever done together with `SetConnection`, so perhaps it can just be passed in that function?

Reply:
Also, it doesn't look like this class even needs to hold on to this pointer, it only needs to pass it to the connection handler and view coordinator.